docs/readthedocs/usersguide.md

```diff
@@ -88,39 +85,39 @@ Using IntelliJ IDEA
 If you are using IntelliJ IDEA as your IDE, this is how setting the command line argument would look like:
 
 
-#. 
-   Find the ``vn_simona.conf`` file in your project directory and copy its path from the content root
+#.
+Find the ``vn_simona.conf`` file in your project directory and copy its path from the content root
+
+.. figure:: ./images/usersguide/copy-path.png
+
+
+#.
+Go to the window "Select Run/Debug Configuration" and select "Edit Configurations"
 
-   .. figure:: ./images/usersguide/copy-path.png
 
+.. figure:: ./images/usersguide/edit-conf.png
 
-#. 
-   Go to the window "Select Run/Debug Configuration" and select "Edit Configurations"
 
-    
-   .. figure:: ./images/usersguide/edit-conf.png
+#.
+Enter ``--config={path from the content root of config file}``
+
+.. note::
```

Comment:
Note doesn't work

Reply:
Replaced the old note format with the markdown format.